var/spack/repos/builtin/packages/hip-rocclr/package.py

```diff
@@ -25,7 +25,7 @@ def url_for_version(self, version):
         url = "https://github.com/ROCm-Developer-Tools/ROCclr/archive/rocm-{0}.tar.gz"
         return url.format(version)
 
-    version('master', branch='main')
+    version('master', branch='master')
```

Comment:
The main branch on here hasn't been touched in 5 years, the master branch in 2, it looks like the only active branch is the default branch, which is develop.  Why add what looks like a dead branch instead of the one that's in use?

Reply:
Hi @trws , have done the necessary changes, Could you please review